tests/regression/test_accounting.py

```diff
@@ -166,12 +161,6 @@ def test_accounting_negative_cl_rebase(accounting_oracle: Contract, lido: Contra
         eth_distributed_event["preCLBalance"] + rebase_amount == eth_distributed_event["postCLBalance"]
     ), "ETHDistributed: CL balance differs from expected"
 
-    post_ttl_shares_event = _first_event(tx, PostTotalShares)
```

Comment:
because LegacyOracle has been disconnected from receiving updates, see: https://research.lido.fi/t/steth-on-optimism-upgrade-announcement-and-action-plan/8474#p-17840-legacyoracle-deprecation-notice-6